gallery/visualtests/nxToast.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+const { setupBrowser } = require('./testUtils');
+
+describe('NxToast', function() {
+
+  const viewportSize = { width: 1366, height: 1000 };
+
+  const launchToastsFromPage = (wait, waitAndGetElements) => async () => {
```

Comment:
Now that all of the other repetition is cleaned up, this can be moved back into the `describe` block with the `setupBrowser` call so that `wait` and `waitAndGetElements` are available in the closure scope and you don't have to pass them in.

Reply:
> Now that all of the other repetition is cleaned up, this can be moved back into the `describe` block with the `setupBrowser` call so that `wait` and `waitAndGetElements` are available in the closure scope and you don't have to pass them in.

moved back in here: [6ba2c0c](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/6ba2c0cf3d74e6ad981a88e2904665e0c81b2fda)